src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {

```

Comment:
Lottery should not extend Ball. Instead, create a separate Ball class with color and number fields.